index.html

```diff
@@ -1,686 +0,0 @@
-<!DOCTYPE html>
-<html lang="pt-br">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="./css/reset.css">
-    <link rel="stylesheet" href="./css/style.css">
-    <link rel="icon" type="imagem/png" href="/imagens/icone-nav.png"/>
-    <link  rel="preconnect" href="https://fonts.gpost-storiesic.com">
-    <link  href ="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel ="stylesheet">
-    <title>Instagram</title>
-
-</head>
-<body>
-
-  <header class="cabecalho">
-    <div class="topo">
-      <div class="logo">
```

Comment:
Uma boa prática na nomeação de classes é usar todos os termos em inglês. Por exemplo, você tem uma <div class="cabecalho">, e logo em seguida, dentro dela tem uma <div class="topo"/>.
Não é errado fazer isso, só é mais conveniente manter um estilo padrão pra nomear. Isso ajuda bastante pra quando você for trabalhar com uma equipe de devs, evitando aquela sensação de insegurança sobre "como nomear esse cara", tendo um padrão estabelecido, essa dúvida nunca vai aparecer.
Resumindo, é recomendável usar nomes em inglês nas classes css.